docs/features/pastefromgdocs/README.md

```diff
@@ -0,0 +1,74 @@
+---
+category: inserting-content
+order: 50
+url: features/pastefromgdocs
+menu-title: Paste from Google Docs
+meta-title-short: Paste from Google Docs
+---
+<!--
+Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md.
+-->
+
+# Pasting Content from Google Docs
+
+<info-box info="">
+    This feature is provided through the <a href="https://ckeditor.com/cke4/addon/pastefromgdocs">Paste from Google Docs</a> plugin that is included in the Standard and Full presets available from the official CKEditor <a href="https://ckeditor.com/ckeditor-4/download/">Download</a> site. You can also {@link guide/dev/plugins/README add it to your custom build} with <a href="https://ckeditor.com/cke4/builder">online builder</a>.
+</info-box>
+
+The [Paste from Google Docs](https://ckeditor.com/cke4/addon/pastefromgdocs) plugin allows you to paste content from Google Docs and maintain original content structure and formatting.
+
+When enabled, it automatically detects Google Docs content and transforms its structure and formatting to clean HTML.
+
+## Supported Features
+
+Paste from Google Docs retains the following formatting:
+
+* Text formatting
+    * Text and background colors
+    * Font name, style and size
+    * Basic formatting (bold, italic, underline)
+    * Font effects (strikethrough, superscript, subscript)
+    * Heading levels
+    * Text alignment
+* Lists
+    * Numbered and bulleted lists
+    * Multilevel lists
+    * Different numbering formats (Roman, decimal, alphanumeric)
+    * Custom start number (e.g. you can start the list from number 4)
+* Tables
+    * Borders and shading
+    * Cell size (width and height)
+    * Cell alignment
+    * Text formatting (as listed above)
```

Comment:
I think there is no need to duplicate this information, since text formatting is supported in general (so when mentioning in tables context it should be also mentioned in lists context etc).